tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/matterusers.controller.js

```diff
@@ -322,7 +322,13 @@
                 cm.assignPermissionTeams.push(assignedTeam);
                 var userDetails = {};
                 userDetails.userName = userNameValue + "(" + userEmailValue + ");";
-                userDetails.userRole = roles[i];
+                //userDetails.userRole = roles[i];
+                if (roles == undefined || roles.length === 0) {

```

Comment:
@premchandpl always assign assignedTeam.assignedRole.name.  The is no need to check as the check already performed above prior to adding role info into assignedTeam.assignedrole.. see the line number 294